src/praisonai-bot/praisonai_bot/bots/_dlq.py

```
@@ -522,6 +538,55 @@ async def replay(
 
         return success_count
 
+    async def redeliver(
+        self,
+        send: Callable[[OutboundDLQEntry], Awaitable[Any]],
```

Ruff (0.16.1)

[error] Undefined name `Any`

(F821)